src/store/model/bol.ts

```diff
@@ -0,0 +1,43 @@
+import {Store} from './store';
+
+export const Bol: Store = {
+	labels: {
+		captcha: {
+			container: 'body',
+			text: ['voer de karakters in die u hieronder ziet']
+		},
+		inStock: {
+			container: '.buy-block',
+			text: ['Op voorraad']
+		},
+		maxPrice: {
+			container: 'span[class*="PriceString"]',
```

Comment:
```suggestion
			container: '.price-block__price',
```

or

```suggestion
			container: '.promo-price',
```

But not sure how reliable that is. `promo` makes it seem like it's temporary.